tests/playwright-tests/tests/admin-and-public/publishPublication.spec.ts

```diff
@@ -69,6 +69,7 @@ test('Verify that user is able to create a release via admin', async ({
 test('Validate that user is able to see the created release in public and title is same as publication name', async () => {
   await homePage.navigateToExploreFindStatisticsPage();
   await findStatisticsPage.navigateToPublicReleasePage(publicationName);
-  const title = await releasePage.pageTitle.textContent();
+  
+  const title = await releasePage.pageTitle(publicationName).innerText();
   await expect(title).toContain(publicationName);
```

Comment:
Can't help but notice the docs discourage the use of `innerText()` -> https://playwright.dev/docs/api/class-locator#locator-inner-text

Does it work if we replace both these lines with something like `await expect(releasePage.pageTitle(publicationName)).toHaveText(publicationName);` ? If it doesn't, we can pass in the `useInnerText` config option like it says in the docs - 
![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/63285990/d74f1e46-7c93-48ba-8741-c5a33ab2d4a7)